src/components/vector/ExportDialog.tsx

```
@@ -92,206 +102,259 @@ export function ExportDialog({ open, onOpenChange }: ExportDialogProps) {
         case 'dot':
-          // Punto - círculo pequeño
-          const dotRadius = actualLength * 0.15;
-          return `<circle id="${id}" cx="${actualLength/2}" cy="0" r="${dotRadius}" fill="${settings.vectorColor}" ${transform} />`;
-        
+          // Punto
+          return `<circle cx="${actualLength / 2}" cy="0" r="${(settings.vectorWidth ?? 2) * 1.5}" fill="${settings.vectorColor ?? '#FFFFFF'}" ${transform}/>`;
+          
         case 'triangle':
           // Triángulo
-          const tipX = actualLength; // Solo en el eje X para que la rotación funcione 
           const angle1 = Math.PI * 0.8; // 144 grados
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)